# The ActuorDetection service returns a list of actuors' states and the ok flag.

#request constants
---
#response constants
bool ok
uint32[] id
int8[] states 
